src/checkbox-group.ts

```diff
@@ -478,7 +478,7 @@ export default class GlideCoreCheckboxGroup
   @state()
   private validityMessage?: string;
 
-  #componentElementRef = createRef<HTMLDivElement>();
+  #componentElementRef = createRef<HTMLElement>();
```

Comment:
`HTMLDivElement` is a bit brittle and doesn't give use any special attributes or properties.